src/lib/components/Snackbar.react.js

```diff
@@ -109,14 +114,15 @@ export default class Snackbar extends Component<Props, State> {
   };
 
   render() {
-    const {action, actionStyles, autoHideDuration, className, id, message, style } = this.props;
+    const {action, actionStyles, autoHideDuration, className, children, id, message, style } = this.props;
 
     return action ? (
       <div id={id}>
         <MuiThemeProvider muiTheme={getMuiTheme(lightBaseTheme)}>
           <MuiSnackbar
             autoHideDuration={autoHideDuration}
             className={className}
+            children={children}
```

Comment:
Is there any need for us to check the values of `message` or `children` before attempting to render them? If the user submits both, ignoring the warning from the docs, what happens?

Reply:
Children will override Message and Actions. I modified the documentation to reflect this as well